### PR TITLE
add second test

### PR DESCRIPTION
**Pull request by eric891224**
added a test for the edge case in which any of the inputs to `add` is not of type `number`

### CODE REVIEW

**Comment by eric891224** [comment]
lgtm

File: tests/unit/mathService.test.ts [hunk 1]
```diff
@@ -9,4 +9,11 @@ describe("Math Service - add", () => {
         const result = math.add(1, 2)
         expect(result).toBe(3)
     })
+
+    it("should throw an error if any of the arguments is not a number", () => {
+        expect(() => math.add(1, "2" as any)).toThrow("Both arguments must be numbers")
+        expect(() => math.add("1" as any, 2)).toThrow("Both arguments must be numbers")
+        expect(() => math.add("1" as any, "2" as any)).toThrow("Both arguments must be numbers")
+        expect(() => math.add(1, null as any)).toThrow("Both arguments must be numbers")
```
Comment (eric891224) [inline tests/unit/mathService.test.ts:17]: should add another test that checks when the first argument is null
